http/src/main/java/io/micronaut/http/filter/FilterOrderProvider.java

```diff
@@ -1,29 +0,0 @@
-/*
- * Copyright 2017-2020 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.http.filter;
-
-import io.micronaut.core.order.Ordered;
-
-/**
- * Describes a bean that contains an order to define the
- * order of a client or server filter.
- *
- * @author James Kleeh
- * @since 1.0
- */
-public interface FilterOrderProvider extends Ordered {
```

Comment:
did this exist in the 3.x branch? Probably need to deprecate it there if so

Reply:
yes but it was unused. im not sure what it was supposed to do in the first place